scripts/run_vm.sh

```diff
@@ -6,19 +6,20 @@ SCRIPT=$(realpath -s "${0}")
 SCRIPTS_PATH=$(dirname "${SCRIPT}")
 TESTS_PATH=$(realpath -s "${SCRIPTS_PATH}/../tests")
 
+: "${ELMNTL_PREFIX:=}" 
```

Comment:
I added a prefix to the files so we could consider at some point running in parallel multiple tests on the same machine without overlapping files.